src/solvers/refinement/string_constraint.h

```diff
@@ -204,6 +208,21 @@ class string_not_contains_constraintt: public exprt
   }
 };
 
+inline static std::string from_expr(
+  const namespacet &ns,
+  const irep_idt &identifier,
+  const string_not_contains_constraintt &expr)
```

Comment:
Can you document what this function does. 
Also `from_expr` does not really describe what this function does, though I know this is to stay coherent with what's already in cbmc. 
Also this maybe overload `from_expr` defined on `exprt` from which `string_not_contains_constraintt` inherits which may be confusing (we may expect one to be called when it's the other).